.babelrc

```diff
@@ -1,4 +1,6 @@
 {
-  "presets": ["es2015", "react", "stage-0"],
-  "plugins": ["add-module-exports"]
+  "presets": ["module:@skpm/babel-preset", "@babel/preset-flow"],
+  "plugins": ["@babel/plugin-proposal-class-properties", ["@babel/plugin-transform-modules-commonjs", {
+    "noInterop": true
+  }]]
```

Comment:
The plugin is required to export commonjs modules. It’s not the default with Babel 7 anymore. I chose to add the noInterop option because it reduces the bundle size a lot. But it also means that I needed to change a bunch of imports in order to make them compatible. For example `react` doesn’t export a `default` so you need to `import * as react` from it. This is actually the correct way to import a commonjs package (typescript enforces it).

Reply:
I had no idea that [React doesn't ship an es module distro](https://github.com/facebook/react/issues/10021) 🙀

I suppose this all makes sense internally.

Externally, should we be updating our docs and examples. While it may be correct, wonder if it may cause confusion. And maybe I am incorrect here, but I would expect most people are used to seeing `import React from 'react'`.

Reply:
yeah happy to change the docs back